extensions/amp-analytics/0.1/visibility-model.js

```diff
@@ -71,7 +71,9 @@ export class VisibilityModel {
     });
 
     this.eventPromise_.then(() => {
-      this.onTriggerObservable_.fire();
+      if (this.onTriggerObservable_) {
+        this.onTriggerObservable_.fire();
+      }
```

Comment:
How about we dev().warn in an else preferably with info that may help debug the problem (e.g. page url assuming it isn't already present)